tests/php/LeftAndMainTest.php

```diff
@@ -175,4 +161,22 @@ public function testCanView()
 
         $this->session()->set('loggedInAs', null);
     }
+
+    /**
+     * Test that getHelpLinks transforms $help_links into the correct format
+     */
+    public function testGetHelpLinks()
+    {
+        Config::inst()->set(LeftAndMain::class, 'help_links', [
+            'SilverStripe' => 'www.silverstripe.org',
+        ]);
+
+        $helpLinks = LeftAndMain::singleton()->getHelpLinks();
+        $this->assertCount(1, $helpLinks);
+
+        $silverstripeLink = $helpLinks->first();
+
+        $this->assertEquals($silverstripeLink['Title'], 'SilverStripe');
+        $this->assertEquals($silverstripeLink['URL'], 'www.silverstripe.org');
```

Comment:
Can you please flip the arguments? The expected result should be the first argument